### DISCUSSION TITLE
Docker norstella/oxalis-as4 - Include SSHD

### DISCUSSION THREAD

**Discussion by francescodiperna** [OxalisCommunity/Oxalis-AS4#169] [Ideas]
Hi,
I'm using the image "norstella/oxalis-as4" in hub.docker.com.
I would like to ask to the developer if is possible to include the SSHD service.
Thanks

**Comment by aaron-kumar** [comment]
We will look into it

**Comment by alfchristensen** [comment]
@francescodiperna if you are still interested here is the code for Alpine Linux, which is used for version including the current norstella/oxalis:latest (5.5.0)

Install and enable SSH - Alpine
`RUN apk add openssh && echo "root:Docker!" | chpasswd && cd /etc/ssh/ && ssh-keygen -A`

If you want it to run on Azure, please follow this guide:
https://learn.microsoft.com/en-us/azure/app-service/configure-custom-container?pivots=container-linux&tabs=debian#enable-ssh

**Reply by francescodiperna** [reply]
Thanks
